.github/workflows/build-tests.yml

```diff
@@ -0,0 +1,56 @@
+name: Continuous Integration
+on: 
+  push:
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/python@master
+      with:
+        command: test
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      continue-on-error: true
+
+    - run: docker build --target test -t app-tests .
+    - run: docker run -e SECRET_KEY=${{ secrets.SECRET_KEY }} -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_API_TOKEN=${{ secrets.TRELLO_API_TOKEN }} -e TRELLO_BOARD_ID=${{ secrets.TRELLO_BOARD_ID }} -e TRELLO_DEFAULT_LIST_ID=${{ secrets.TRELLO_DEFAULT_LIST_ID }} app-tests
```

Comment:
Definitely minor, but in case it's of interest; this command is obviously quite long; one way we can split it up is by taking advantage of the fact that if we tell Docker an env variable exists but don't tell it where to find it, it assumes the same name on host as in the container. Which means if `$SECRET_KEY` already exists, `-e SECRET_KEY` is enough. So we can split this out into more like:
```suggestion
    - run: docker run -e SECRET_KEY -e TRELLO_API_KEY -e TRELLO_API_TOKEN -e TRELLO_BOARD_ID -e TRELLO_DEFAULT_LIST_ID app-tests
      env:
        SECRET_KEY: ${{ secrets.SECRET_KEY }}
        TRELLO_API_KEY: ${{ secrets.TRELLO_API_KEY }}
        TRELLO_API_TOKEN: ${{ secrets.TRELLO_API_TOKEN }}
        TRELLO_BOARD_ID : ${{ secrets.TRELLO_BOARD_ID }}
        TRELLO_DEFAULT_LIST_ID : ${{ secrets.TRELLO_DEFAULT_LIST_ID }}
        ....
```

Whether you think that's cleaner or not... 🤷

Reply:
Definitely more readable at the very least, this is good to know thanks!